bitnami/kube-prometheus/templates/prometheus/additionalScrapeJobs.yaml

```
@@ -0,0 +1,10 @@
+{{- if (and .Values.prometheus.additionalScrapeConfigs.enabled (eq .Values.prometheus.additionalScrapeConfigs.type "internal") ) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: additional-scrape-jobs-{{ template "kube-prometheus.prometheus.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "kube-prometheus.prometheus.labels" . | nindent 4 }}
+data:
+  scrape-jobs.yaml: {{ toYaml .Values.prometheus.additionalScrapeConfigs.internal.jobList | b64enc | quote }}
```

Could we use  "common.tplvalues.render" here? so that we will be able to templatize the job definition.

---

@dani8art Happy to make this change but would need a little guidance on where to plug this in and make sure it's still b64 encoded. something like this?
```suggestion
  scrape-jobs.yaml: {{ include "common.tplvalues.render" ( dict "value" .Values.prometheus.additionalScrapeConfigs.internal.jobList "context" $ ) | b64enc | quote }}
```